putio.py

```diff
@@ -9,6 +9,15 @@
 import requests
 import iso8601
 
+KB = 1024
+MB = 1024 * KB
+
+# Downloads are broken up into ranges of this size
+RANGE_SIZE = 10 * MB
```

Comment:
The range size is chosen for safety reasons. At the end of each range the written data is flushed. It also allows me to have a log of the progress during a very long file download. It will start up from where it left off. Happy to remove the range and just try the whole leftover chunk if you think that's better.